src/lib/abacus/index.ts

```diff
@@ -89,22 +89,38 @@ class AbacusStateManager {
       this.abacusFormatter
     );
 
-    const appConfigs = new AbacusAppConfig(
-      false, // subscribeToCandles
-      true, // loadRemote
-      import.meta.env.MODE === 'development' && import.meta.env.VITE_ENABLE_ABACUS_LOGGING // enableLogger
-    );
-    appConfigs.squidVersion = AbacusAppConfig.SquidVersion.V2;
-
-    this.stateManager = new AsyncAbacusStateManager(
-      '',
-      CURRENT_ABACUS_DEPLOYMENT,
-      appConfigs,
-      ioImplementations,
-      uiImplementations,
-      // @ts-ignore
-      this.stateNotifier
-    );
+    if (import.meta.env.MODE === 'development') {
+      const appConfigs = AbacusAppConfigV2.Companion.forAppWithIsolatedMargins;
+      appConfigs.enableLogger = import.meta.env.MODE === 'development' && import.meta.env.VITE_ENABLE_ABACUS_LOGGING;
+  
+      this.stateManager = new AsyncAbacusStateManagerV2(
+        '',
+        CURRENT_ABACUS_DEPLOYMENT,
+        appConfigs,
+        ioImplementations,
+        uiImplementations,
+        // @ts-ignore
+        this.stateNotifier
+      );
+    } else {
+      const appConfigs = new AbacusAppConfig(
+        false, // subscribeToCandles
+        true, // loadRemote
+        import.meta.env.MODE === 'development' && import.meta.env.VITE_ENABLE_ABACUS_LOGGING // enableLogger
```

Comment:
[q] maybe i'm just having trouble reading this but would this ever happen? looks to me like this is in the `else` block of `import.meta.env.MODE === 'development'` so it can't happen

Reply:
@yogurtandjam is right here

Reply:
Yeah the intention was to leave this config as-is for clarity, in case we want to change things around later.